app/controllers/spree/admin/locales_controller_decorator.rb

```diff
@@ -1,15 +1,25 @@
 # frozen_string_literal: true
 
-Spree::Admin::LocaleController.class_eval do
-  before_action :update_i18n_settings, only: :update
+module Spree
+  module Admin
+    module LocalControllerDecorator
```

Comment:
This should be LocalesControllerDecorator.

Reply:
This is probably telling us that this file not needed anymore? 😄

Reply:
Nevermind, we are still using it since we prepend it with `self`.